torch/jit/__init__.py

```diff
@@ -346,8 +340,12 @@ def __process_args(self, args):
 
     # NB: In principle, there could also be a 'raw' version of this compiler,
     # but since the logic is so complicated, testing code wouldn't benefit much
-    def __new_forward(self, *args):
-        if _JIT_DISABLE or not self.__enabled:
+    def __new_forward(self, *args, **kwargs):
+        enabled = kwargs.pop("enabled", True)
```

Comment:
This is really ugly. We shouldn't add any arguments to `forward`, especially `enabled=False`. Think that the complile annotation is very far from the actual use of the module, and `module(x, enabled=False)` looks very weird

Reply:
The "enabled" is an accident. 

`_force_trace` is intended to be a private kwarg. If we really want to do it differently, I can alternately add a method to delete a trace associated with an argument.